sonargraphintegration.properties

```diff
@@ -4,20 +4,26 @@ category=External Analyzers
 description=Provides support for Sonargraph 8+'s architecture governance features, accompanied by metrics about cyclic dependencies and other structural aspects.
 homepageUrl=https://github.com/sonargraph/sonar-sonargraph-integration
 devVersion=
-publicVersions=4.1.1
-archivedVersions=1.1.2,1.1.3,1.1.4,2.0.1,2.1.5,2.1.7,3.1.0,4.0.0
+publicVersions=4.1.2
+archivedVersions=1.1.2,1.1.3,1.1.4,2.0.1,2.1.5,2.1.7,3.1.0,4.0.0,4.1.1
 privateVersions=
 
 defaults.mavenGroupId=com.hello2morrow
 defaults.mavenArtifactId=sonar-sonargraph-integration
 
+4.1.2.description=Compatibility with Sonargraph 9.13 and newer
+4.1.2.sqVersions=[7.9,LATEST]
+4.1.2.date=2020-02-05
+4.1.2.changelogUrl=https://github.com/sonargraph/sonar-sonargraph-integration/releases/download/Release-4.1.2/release-notes.txt
+4.1.2.downloadUrl=https://github.com/sonargraph/sonar-sonargraph-integration/releases/download/Release-4.1.2/sonar-sonargraph-integration-4.1.2.jar
+
+#====================
 4.1.1.description=Compatibility with SonarQube 8.1
-4.1.1.sqVersions=[7.9,LATEST]
+4.1.1.sqVersions=[6.7,8.1]
```

Comment:
6.7 is no longer recognized in the Marketplace. Please keep the bottom of this range at 7.9.

Reply:
I thought this does not matter for archived versions?

Reply:
a) we've archived versions of SQ <7.9 so IIRC, this will cause errors in the generation job
b) it started out at 7.9 & compatibility on archived versions is about upgrade prompts in the Marketplace. Since this wasn't originally marked compatible with 6.7, no one on 6.7 should be running the version & _need_ to upgrade from it.

Reply:
Never mind, I thought, since the plugin version 4.1.1 was programmed against the older API, it would be ok to mention it in the archived section. Wasn't aware of the consequences.